website/versioned_sidebars/version-18.4.0-sidebars.json

```diff
@@ -0,0 +1,336 @@
+{
```

Comment:
Based on the current pattern I think this file should be named `version-v18.4.0-sidebars.json`